lambdas/process-journey-step/src/main/java/uk/gov/di/ipv/core/processjourneystep/ProcessJourneyStepHandler.java

```diff
@@ -59,8 +59,8 @@ public ProcessJourneyStepHandler(
 
     @ExcludeFromGeneratedCoverageReport
     public ProcessJourneyStepHandler() throws IOException {
-        this.stateMachine = new StateMachine(new StateMachineInitializer());
         this.configurationService = new ConfigurationService();
+        this.stateMachine = new StateMachine(new StateMachineInitializer(configurationService));
```

Comment:
Does passing in the whole config service help here? Can we just pass in the env var?

Reply:
So yeah we aren't using it now with the multiple yaml files, but I left this in incase we needed to revert back or change to using the CriResponse class again. I can remove this for now though as I guess its all going to be moved into the CriSelector lambda anyways